### PR TITLE
Text Updates

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Brief description of what this PR does, and why it is needed.

Connects #1964 

### Demo

image

image

image

image

image

image

image

image

### Notes

They wanted help links for each of the models in the model selection boxes in Analyze, in the last screenshot. I tried to add them, but clicking the links would also select the model and proceed. It may take some time to figure out how to do it right, so I skipped them for now. I did add them to `base.py` as `modelPackage.help_link` so we can add them in when we can.

## Testing Instructions

 * Check out this branch and run `update`
 * Ensure there are no more `Lorem ipsums` in the app
 * Ensure text lines up with specifications [here](https://docs.google.com/document/d/13HVyNkrtKKtLufBguCZJq0RDabBCTkjL0vT1PCwz8Lg/edit)

### CODE REVIEW

**Comment by mmcfarland** [comment]
From just browsing the screenshot, there's a spacing issue `andor`

screenshot from 2017-07-20 08 10 34

**Comment by arottersman** [comment]
Looks like we might have overlooked [this comment](https://github.com/WikiWatershed/model-my-watershed/issues/1964#issuecomment-316520838)

- The stroud logo and wikiwatershed link haven't been changed

File: src/mmw/js/src/draw/views.js [hunk 1]
```diff
@@ -636,19 +636,31 @@ var DrawAreaView = DrawToolBaseView.extend({
...
             title: 'Draw area',
-            info: 'Lorem ipsum dolor it amet, consectetur',
+            info: 'Free draw an area or place a square kilometer',
             items: [
                 {
                     id: freeDraw,
                     title: 'Free draw',
-                    info: 'Draw a custom area on which you want to perform water quality analysis.',
+                    info: 'Free draw an area of interest polygon, by clicking ' +
+                          'on map and repeatedly clicking at boundary corners. ' +
```
Comment (arottersman) [inline src/mmw/js/src/draw/views.js]: This is what they wrote, but I think it should read `on the map` not just `on map`

File: src/mmw/js/src/draw/views.js [hunk 2]
```diff
@@ -636,19 +636,31 @@ var DrawAreaView = DrawToolBaseView.extend({
...
+                          'Draw Area.</a>',
                     minZoom: 0,
                     directions: 'Draw a boundary.'
                 },
                 {
                     id: squareKm,
                     title: 'Square Km',
-                    info: 'Draw a predefined one kilometer square with a center at the point you choose.',
+                    info: 'Draw a perfect square with one kilometer sides, by ' +
+                          'clicking on map at the square’s center.<br />' +
```
Comment (arottersman) [inline src/mmw/js/src/draw/views.js]: Same as above, `on map` --> `on the map`

File: src/mmw/mmw/settings/layer_settings.py [hunk 3]
```diff
@@ -193,13 +193,14 @@
...
-            'helptext': 'HUC stands for hydrologic unit code. The hydrologic '
-                        'unit hierarchy is indicated by the number of digits in '
-                        'groups of two (such as HUC-2, HUC-4, and HUC-6) within '
-                        'the HUC code. HUC 8 maps the subbasin level, analogous '
-                        'to medium-sized river basins. There are about 2200 '
-                        'nationwide. This tool allows you to pick a predefined '
-                        'HUC-8 to analyze.',
+            'helptext': 'US Geological Survey Hydrologic Units of the '
+                        'eight-digit level (Hydrologic Unit Code 8), averaging'
+                        ' 700 sq. mi. (1,813 km<sup>2</sup>).<br />'
```
Comment (arottersman) [inline src/mmw/mmw/settings/layer_settings.py]: For each of these, consider using a `nbsp` between `1,813` and `km` (or no space at all)

They wrap weird.
screen shot 2017-07-20 at 8 43 07 am

File: src/mmw/mmw/settings/layer_settings.py [hunk 4]
```diff
@@ -228,13 +230,14 @@
...
-            'helptext': 'HUC stands for hydrologic unit code. The hydrologic '
-                        'unit hierarchy is indicated by the number of digits in '
-                        'groups of two (such as HUC-2, HUC-4, and HUC-6) within '
-                        'the HUC code. HUC-12 is a more local sub-watershed level '
-                        'that captures tributary systems. There are about 90,000 '
-                        'nationwide. This tool allows you to pick a predefined '
-                        'HUC-12 to analyze.',
+            'helptext': 'US Geological Survey Hydrologic Units of the '
+                        'twelve-digit level (Hydrologic Unit Code 12),'
+                        'averaging 40 sq. mi. (104 km<sup>2</sup>).<br />'
```
Comment (arottersman) [inline src/mmw/mmw/settings/layer_settings.py]: Need space before `averaging`

screen shot 2017-07-20 at 8 44 50 am

File: src/mmw/js/src/draw/templates/window.html [hunk 5]
```diff
@@ -1,10 +1,15 @@
 <h2>Select Area</h2>
 <p id="draw-pane-header-description">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-    Duis ac pharetra nibh. Nullam gravida massa non egestas
-    pellentesque. More information and help for these tools
-    is available <a href="http://wikiwatershed.org/model/" target="_blank">
-    here</a>.
+    Explore mapped layers, such as streams, land cover, soils, boundaries and
+    observations, using the layer selector in the lower left of map. For more
```
Comment (arottersman) [inline src/mmw/js/src/draw/templates/window.html]: `map` --> `the map`?

File: src/mmw/js/src/draw/views.js (hunk 2 above)
Comment (arottersman) [inline src/mmw/js/src/draw/views.js]: I also find this part of the description a little confusing:  `by clicking on map at the square’s center`. It's referencing the location of the yet undrawn square as though it's already on the map. The action is really `clicking on the map where the square's center should be`, though that doesn't sound any better.

File: src/mmw/js/src/analyze/templates/analyzeDescription.html [hunk 6]
```diff
@@ -2,6 +2,15 @@
...
     {% if source %}
-        Source: {{ source }}
+        <span>Source: {{ source }}</span>
+        {% if helpText %}
+            <a class="help" data-toggle="popover" tabindex="0" data-html="true"
+               data-container="body" role="button" data-content="{{ helpText }}"
+               data-template="<div class='popover'><div class='pull-right'
+                                id='popover-close-button'><i class='fa fa-times' /></div><div
+                                class='popover-content'></div><div class='arrow'></div></div>">
+                <i class="fa fa-question-circle"></i>
```
Comment (arottersman) [inline src/mmw/js/src/analyze/templates/analyzeDescription.html]: Consider having this match the appearance of the mapshed info popover

screen shot 2017-07-20 at 9 13 58 am


(this one)
screen shot 2017-07-20 at 8 54 36 am

**Comment by rajadain** [comment]
Pushed fixups that address the change requests. I was unable to update the link in the help text, because that page now has different text and links as specified in the document:

image

The logo has been updated:

image

Tooltip icon updated to be info rather than question:

image

File: src/mmw/js/src/draw/views.js [hunk 7]
```diff
@@ -636,19 +636,31 @@ var DrawAreaView = DrawToolBaseView.extend({
...
+                          'Draw Area.</a>',
                     minZoom: 0,
                     directions: 'Draw a boundary.'
                 },
                 {
                     id: squareKm,
                     title: 'Square Km',
-                    info: 'Draw a predefined one kilometer square with a center at the point you choose.',
+                    info: 'Draw a perfect square with one kilometer sides, by ' +
+                          'clicking on the map where the square’s center will be.<br />' +
```
Comment (arottersman) [inline src/mmw/js/src/draw/views.js]: 👍

**Pull request merged**